changeEclipse.sh

```diff
@@ -0,0 +1,47 @@
+
+#!/bin/bash
+
+#########################################
+# this script can be used to change the #
+# Eclipse version                       #
+#                                       #
+# Initial version from lhein, 2018      #
+#########################################
+
+# check parameter count and provide help if count doesn't match
+if [ "$#" -ne 2 ]; then
+    echo "Illegal number of parameters"
+    echo "Please specify the old and the new Eclipse name like this:"
+    echo "      changeVersion.sh <oldEclipseName> <newEclipseName>"
+    echo "Example:"
+    echo "      changeVersion.sh photon photon"
```

Comment:
might be best to provide a real exemple changeVersion.sh oxygen photon for instance ;-)

Reply:
I fear I was victim of my own script testing. Will repair :D